client/src/components/QA/QAPhoto.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+export default function QAPhoto({ url }) {
+  return (
+    <Thumbnail
+      thumbnail={url}
+    />
+  );
+}
+
+const Thumbnail = styled.span`
+  height: 75px;
+  width: 75px;
+  background: ${(props) => (props.thumbnail ? `url(${props.thumbnail})` : props.theme.colors.background)};
+  border: 1px solid ${(props) => props.theme.colors.secondary};
```

Comment:
I learnt this from Walter yesterday, you can actually do `${({theme}) => theme.colors.secondary}`